electron/menu.js

```diff
@@ -53,7 +53,11 @@ function buildMenu(appName) {
     },
     {
       label: "View",
-      submenu: [{ role: "reload" }, { role: "forcereload" }],
+      submenu: [
+        { role: "reload" },
+        { role: "forcereload" },
+        { role: "toggleDevTools" },
```

Comment:
This would probably be ok to keep, but I think we don't want it to show for production builds so it's best to take it out if we can't make it dev-only.